pkg/reconciler/events/cloudevent/cloudevent.go

```diff
@@ -121,9 +121,10 @@ func newTektonCloudEventData(ctx context.Context, runObject objectWithCondition)
 	return tektonCloudEventData, nil
 }
 
-// eventForObjectWithCondition creates a new event based for a objectWithCondition,
+// EventForObjectWithCondition creates a new event based for a objectWithCondition,
 // or return an error if not possible.
-func eventForObjectWithCondition(ctx context.Context, runObject objectWithCondition) (*cloudevents.Event, error) {
+// Make public so external clients can import conversion logic
```

Comment:
Thanks @melnikalex and sorry for the delay in responding! lgtm, but would you mind removing this line of the docstring? since all our exported functions can be imported by clients I think it's not necessary

Reply:
done